src/features/mail-service/MailService.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { MailService } from './MailService';
+import { MailData } from './Types';
+
+export class MailServiceController {
+  private mailService: MailService;
+
+  constructor() {
+    this.mailService = new MailService();
+  }
+
+  public async sendMail() {
+    const mailData: MailData = {
+      to: 'anjireddy12382@gmail.com',
+      subject: 'Welcome to Our Service',
```

Comment:
We don't hardcode values here like `to` variable.
`to` value should come from req.body